examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -43,6 +45,8 @@ import {
 const TRANSITION_DURAITON = 4000;
 const DEFAULT_NORMALS_GAP = 30; // Gap for normals visualisation to avoid mess on the screen.
 const NORMALS_COLOR = [255, 0, 0];
+const TEXTURE_CHECKER_URL =
```

Comment:
What is this? A texture? Maybe call it UV_DEBUG_TEXTURE_URL? 

Please name the constant, variables and options consistently.

I recommend we add the texture to our repo and load it via raw github URLs.

Just need to make sure the license is OK if we copy something existing.

Reply:
It's free https://www.facebook.com/maurus.loeffel/posts/10223606160928053 . I plan to compress it and move in our repo